devsetup/Makefile

```diff
@@ -4,6 +4,10 @@ PULL_SECRET  ?= ${PWD}/pull-secret.txt
 CRC_DEFAULT_NETWORK_IP ?= 192.168.122.10
 EDPM_COMPUTE_SUFFIX ?= 0
 EDPM_TOTAL_NODES ?= 1
+EDPM_COMPUTE_VCPUS ?= 8
+EDPM_COMPUTE_RAM ?= 20
+EDPM_COMPUTE_DISK_SIZE ?= 70
```

Comment:
it would change the default for edpm nodes: https://github.com/openstack-k8s-operators/install_yamls/blob/main/devsetup/scripts/gen-edpm-compute-node.sh#L23-L30
I believe this might break CI as it is requiring more resources

Reply:
I'm not sure if it still would change those, looks the same now?

Reply:
Does your concern still apply?

Reply:
yes, it does!
We have different defaults for EDPM and adoption workflows:
https://github.com/openstack-k8s-operators/install_yamls/blob/main/devsetup/scripts/gen-edpm-compute-node.sh#L28-L36

Reply:
for example, you are making EDPM_COMPUTE_DISK_SIZE=70 for both adoption and EDPM dev env,
but the EDPM dev env default is currently 20.
Basically dropping these lines would fix it, as the defaults are set here already: https://github.com/openstack-k8s-operators/install_yamls/blob/main/devsetup/scripts/gen-edpm-compute-node.sh#L28-L36

Reply:
thanks!
I guess, that's done now?